### DISCUSSION TITLE
Platform field specifications

### DISCUSSION THREAD

**Discussion by lduf** [org-SCAN/website#159] [Ressource]
[ICRC-INSA-PlatformFieldSpecification.xlsx](https://github.com/TC-netw4ppl/website/files/7459402/ICRC-INSA-PlatformFieldSpecification.xlsx)

**Comment by mathisfchx** [comment]
# Conclusions :
-Personn id uniques, perte de l'indentification par evenement (car plusieurs ev possible par personn)
-Ev id voir excel aaa-2021-_ _ _ _ _ (5 unique ? sequential numbers, ils sont rattachés au ISO code ? ou unique standardisés).
-Source : source type, creator, quality, unique source id (si but multiorganisation revoir la portée de la prise d'informations)

**Comment by mathisfchx** [comment]
revoir le actor id proposé, non fonctionnel et pas pratique

**Reply by mathisfchx** [reply]
VALIDE A REVOIR

**Comment by mathisfchx** [comment]
p3 sources
